static/js/components/dashboard/CourseGrade_test.js

```diff
@@ -4,23 +4,30 @@ import { shallow } from 'enzyme';
 import { assert } from 'chai';
 import _ from 'lodash';
 
+import { edxLinkBase } from './CourseDescription';
 import CourseGrade from './CourseGrade';
 
 import { findAndCloneCourse } from '../../util/test_utils';
 
 describe('CourseGrade', () => {
   let gradeTypes = {
-    current_grade: "Current grade",
+    current_grade: "Course Progress",
     final_grade: "Final grade"
   };
 
   _.forEach(gradeTypes, function(expectedLabel: string, gradeKey: string) {
     it(`shows the ${expectedLabel} for a course`, () => {
       let course = findAndCloneCourse(course => course.runs.length > 0);
       let grade = '50';
+      let progressUrl = `${edxLinkBase}${course.runs[0].course_id}/progress`;
       course.runs[0][gradeKey] = grade;
 
+
       const wrapper = shallow(<CourseGrade courseRun={course.runs[0]} />);
+      assert.equal(
+        wrapper.find(".course-grade .number").childAt(0).html(),
+        `<a href="${progressUrl}" target="_blank">${grade}%</a>`
+      );
```

Comment:
the test for the grade text is already done at line 31. can you just change this into a test for the href value for the link? if you want, you can test the target value too. it would make sense to assign `wrapper.find(".course-grade .number")` to a variable since this test and the test at line 31 both need it